### PR TITLE
Test with scope=host

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/71455) / small_tests / b841bd047410989ed6bb0226dc83d44c33ff6241
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3257/71455/small_tests.24.0.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3257/71455/small_tests.24.0.4/small/ct_run.mongooseim@localhost.2021-09-09_22.50.28/lib.mongooseim.logs/run.2021-09-09_22.50.28/suite.log.html)

**Comment by vkatsuba** [comment]
[GHA](https://github.com/esl/MongooseIM/actions/runs/1219115140)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/71582) / small_tests / 9d3a1127feda0a78e88e2f661d6844568b4760d9
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3257/71582/small_tests.24.0.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3257/71582/small_tests.24.0.4/small/ct_run.mongooseim@localhost.2021-09-10_07.31.49/lib.mongooseim.logs/run.2021-09-10_07.31.49/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/71795) / small_tests / 8c21c37e6d3fc2f545f16a1b4613718ee73ea5f0
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3257/71795/small_tests.24.0.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3257/71795/small_tests.24.0.4/small/ct_run.mongooseim@localhost.2021-09-10_11.25.15/lib.mongooseim.logs/run.2021-09-10_11.25.15/suite.log.html)

File: big_tests/test.config [hunk 1]
```diff
@@ -227,20 +227,21 @@
         {auth_method, <<"SASL-ANON">>}]}
                      ]}.
 
+%% All options "scope" which have value "host" should be tested with option "global" locally before creating PR.
```
Comment (chrzaszcz) [inline big_tests/test.config:230]: I think we could simply test it with `scope = global` in some of the presets below, that's what we discussed at the daily.

File: big_tests/test.config [hunk 2]
```diff
@@ -227,20 +227,21 @@
...
 {presets,
  [{toml,
    [
     {internal_mnesia,
      %% dbs variable is used by ./tools/test_runner/presets_to_dbs.sh script
      [{dbs, [redis, minio]},
       {outgoing_pools, "[outgoing_pools.redis.global_distrib]
-  scope = \"global\"
+  scope = \"host\"
```
Comment (chrzaszcz) [inline big_tests/test.config:238]:
```suggestion
  scope = \"global\"
```

**Review by chrzaszcz — COMMENTED** [review]
I added some comments, maybe now it is easier to understand what I proposed.

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/72266) / small_tests / 3f1c33e16bbfc9bd5d63db3ea8f8af242caec017
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3257/72266/small_tests.24.0.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3257/72266/small_tests.24.0.4/small/ct_run.mongooseim@localhost.2021-09-13_12.33.19/lib.mongooseim.logs/run.2021-09-13_12.33.19/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/72290) / small_tests / 87d65f5244e1d4060dd64a11a50b29c322781e50
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3257/72290/small_tests.24.0.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3257/72290/small_tests.24.0.4/small/ct_run.mongooseim@localhost.2021-09-13_12.59.37/lib.mongooseim.logs/run.2021-09-13_12.59.38/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks good!

**Pull request merged**